stdlib/buffer.md

```diff
@@ -15,9 +15,19 @@ No other changes yet.
 include "buffer"
 ```
 
+## Types
+
+Type declarations included in the Buffer module.
+
+### Buffer.**Buffer**
+
+```grain
+type Buffer
```

Comment:
I wonder if we want to show `abstract type Buffer` so it's clear that it's an abstract type and not just missing information.

Reply:
Unrelated to this PR. Please open an issue since this wasn't changed in the abstract PR.

Reply:
This was just a thought.